docs/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.embeddingsgooglevertex.md

```diff
@@ -0,0 +1,38 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+title: Embeddings Google Vertex node documentation
+description: Learn how to use the Embeddings Google Vertex node in n8n. Follow technical documentation to integrate Embeddings Google Gemini node into your workflows.
+contentType: [integration, reference]
+priority: medium
+---
+
+# Embeddings Google Vertex node
+
+Use the Embeddings Google Vertex node to generate [embeddings](/glossary.md#ai-embedding) for a given text.
+
+On this page, you'll find the node parameters for the Embeddings Google Vertex node, and links to more resources.
+
+/// note | Credentials
+You can find authentication information for this node [here](/integrations/builtin/credentials/google/service-account.md).
+///
+
+--8<-- "\_snippets/integrations/builtin/cluster-nodes/sub-node-expression-resolution.md"
+
+## Node parameters
+
+- **Model**: Select the model to use to generate the embedding.
+
+Learn more about available embedding models in [Google VertexAI embeddings API documentation](https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/text-embeddings-api){:target=\_blank .external-link}.
```

Comment:
```suggestion
Learn more about available embedding models in [Google VertexAI embeddings API documentation](https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/text-embeddings-api).
```
Not a mistake, but we are deprecating these external link attributes like this gradually since this is now handled by a different mechanism.